app/controllers/api/v1/tickets_controller.rb

```
@@ -1,240 +1,240 @@
+                end
+            
+                # Filter tickets by category if provided
+                if params[:category].present?
+                    @tickets = @tickets.where(category: params[:category])
+                end
+            
+                render json: @tickets
+                else
+                render json: { error: 'organization_id is required.' }, status: :unprocessable_entity
+                end
+            end
```

RuboCop (1.69.1)

[convention] Assignment Branch Condition size for index is too high. [<6, 42, 9> 43.37/23]

(Metrics/AbcSize)